thingif/src/main/java/com/kii/thingif/command/CommandForm.java

```diff
@@ -22,23 +20,18 @@
  * <br><br>
  * Mandatory data are followings:
  * <ul>
- * <li>Name of a schema</li>
- * <li>Version of a schema</li>
  * <li>List of actions</li>
  * </ul>
  * Optional data are followings:
  * <ul>
- * <li>Title of a schema</li>
- * <li>Description of a schema</li>
- * <li>meta data of a schema</li>
+ * <li>Title of a command</li>
+ * <li>Description of a command</li>
+ * <li>meta data of a command</li>
  * </ul>
  */
 public final class CommandForm implements Parcelable {
 
-    @SerializedName("schema")
-    private final @NonNull String schemaName;
-    private final int schemaVersion;
-    private final @NonNull List<Action> actions;
+    private final @NonNull List actions;
```

Comment:
I think that generic without specifying type is not good manner.

I think that introducing interface and implementing the interface at `Action` and `TraitActions` is resolve the issue. What do you think?